### DISCUSSION TITLE
CRNN_VGG16_BN tensorflow model required

### DISCUSSION THREAD

**Discussion by Shiva-Ai4m** [mindee/doctr#1244] [General]
I require crnn_vgg16_bn model file in tensorflow for other purpose please can anyone help in this @felixT2K

**Comment by felixT2K** [comment]
Hi @Shivashankarar you can download the model weights from:
`https://doctr-static.mindee.com/models?id=v0.3.0/crnn_vgg16_bn-76b7f2c6.zip&src=0`
